src/components/Radio/Radio.js

```diff
@@ -19,7 +19,7 @@ class Radio extends React.PureComponent {
     /**
      * The value of the radio; a string that describes what the value should be if checked=true
      */
-    value: PropTypes.string.isRequired,
+    value: PropTypes.any.isRequired,
```

Comment:
What use case would not be `string` here? This prop is passed through to the `<input>` element, which only supports primitive attributes for value.

Reply:
There is a case where we need to pass a `bool` value, which is forbidden here

Reply:
@antkoval A boolean value is not a valid option for the `input` tag's `value` attribute. https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement

Reply:
So the use case is that you have two radio buttons where one is "true" and one is "false"? That seems like a good scenario for a Toggle instead.

If you really need it, convert the values to strings. We don't want to supply invalid attributes to html elements.

Reply:
revert these changes then